api-report/container-runtime.api.md

```diff
@@ -493,6 +494,10 @@ export interface ISummarizerRuntime extends IConnectableRuntime {
     disposeFn?(): void;
     // (undocumented)
     readonly logger: ITelemetryLogger;
+    // (undocumented)
+    off?(event: "op", listener: (op: ISequencedDocumentMessage, runtimeMessage?: boolean) => void): this;
```

Comment:
Let's open a task to make these non-optional in some future.

Reply:
[AB#4018](https://dev.azure.com/fluidframework/internal/_workitems/edit/4018)